include/swift/Basic/Malloc.h

```diff
@@ -44,4 +44,4 @@ inline void AlignedFree(void *p) {
   
 }
 
-#endif
+#endif /* __SWIFT_MALLOC_H__ */
```

Comment:
Also it should be using //, not /*.

Please see:

http://llvm.org/docs/CodingStandards.html#comment-formatting